cmd/service.go

```diff
@@ -102,7 +102,12 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			// Extra space will prevent showing "^CExiting" and rather show "^C Exiting" instead.
+			log.Println(" Exiting")
+			go func() {
+				time.Sleep(time.Millisecond * 600)
+				log.Println("Waiting for active connections to terminate, press ^C again to exit immediately.")
```

Comment:
Alternatively, `Waiting for active connections to terminate - press Ctrl+C to quit immediately` would also be fine.